docs/development/docs.rst

```diff
@@ -10,27 +10,84 @@ The docs are kept in the ``docs/`` directory at the top of the source tree.
    However, if there is going to be periodic/bigger contributions, it may be a
    good idea to install the Sphinx requirements to build our docs.
 
+Contributing through the Github UI
+----------------------------------
+
+If you're making small changes to the documentation,
+you can verify those changes through the documentation generated when you open a PR and can be accessed using the Github UI.
+
+#. click the checkmark next to your commit and it will expand to have multiple options
+#. right-click the "details" link next to the "docs/readthedocs.org:docs" item
+#. navigate to the section of the documentation you worked on to verify your changes
+
+Contributing from your local machine
+------------------------------------
+
+.. note::
+
+    As of April 2021, you must use python 3.7 or lower to generate the documentation.
+
+If you're making large changes to the documentation,
+you may want to verify those changes locally before pushing upstream.
+
+#. clone the `readthedocs.org` repository:
+
+   .. prompt:: bash
+
+      git clone --recurse-submodules https://github.com/readthedocs/readthedocs.org/
+
+#. install documentation requirements
+
+   Run the `tox` command below to install the requirements.
+   More information about the `docs` option can be found on :ref:`development tests <development/tests:Testing>` section.
+
+   .. prompt:: bash
+
+      cd readthedocs.org
+      tox -e docs
+
+   Alternatively, you can try running the following commands if the `tox` command does not work for you.
```

Comment:
After fixing my local setup and the tests, this step isn't needed. Do you think others will run into this or should I remove it?

Reply:
yeah, there is no need to run tox here